client/app/bundles/shared/components/Resource.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+import shortid from 'shortid';
+import css from './Resource.scss';
+import Tag from './Tag';
+
+type Props = {
+  tags: [],
+  tagged?: boolean,
+  link?: string,
+  name?: string,
+  author?: string,
+};
+
+export default class Resource extends React.Component<Props> {
+  render() {
+    const { tags, link, name, author, tagged } = this.props;
+    const resources = tagged ? tags.map(tag =>
```

Comment:
This looks like two different components. One could be AuthorResource and the other TagResource. I would advise against having two different components inside a single component name, especially if they have different props requirements.